.github/workflows/sonarcloud.yml

```
@@ -0,0 +1,188 @@
+            -Dsonar.organization=learnathon-by-geeky-solutions
+            -Dsonar.host.url=https://sonarcloud.io
+            -Dsonar.sources=${{ steps.sonar-params.outputs.sources }}
+            -Dsonar.tests=${{ steps.sonar-params.outputs.tests }}
+            -Dsonar.java.binaries=${{ steps.sonar-params.outputs.binaries }}
+            -Dsonar.java.test.binaries=${{ steps.sonar-params.outputs.test_binaries }}
+            -Dsonar.coverage.jacoco.xmlReportPaths=${{ steps.sonar-params.outputs.jacoco_reports }}
+        env:
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+#added a comment to test the sonarcloud.yml file
```

YAMLlint (1.35.1)

[warning] missing starting space in comment

(comments)